pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -411,3 +416,29 @@ func validateCloneVolumeSnapshotSupportVMSnapshotContent(snapshotContents *v1alp
 
 	return result
 }
+
+func validateNewMacAddresses(vmClone *clonev1alpha1.VirtualMachineClone) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+
+	if vmClone.Spec.NewMacAddresses != nil {
```

Comment:
nit: please reverse the check and use an earlier return. In this way, we can avoid unnecessary nesting level

Reply:
I think we should valide all the MacAddresses.

Reply:
done